index.css

```diff
@@ -87,3 +87,17 @@ button {
   box-shadow: var(--box-shadow-lg);
   transition: all 0.3s ease;
 }
+
+.mobile-show-logo {
+  display: none;
+}
+
+@media screen and (max-width: 640px) {
+  .mobile-hide-logo {
+    display: none;
+  }
+
+  .mobile-show-logo {
+    display: block;
+  }
+}
```

Comment:
These CSS changes effectively add responsive design elements, improving the user experience on mobile devices.

- Good use of media queries to handle responsive design.
- Consider how descriptive class names can make the purpose of CSS classes clearer.